tests/msc2403_test.go

```diff
@@ -40,6 +45,20 @@ func TestKnocking(t *testing.T) {
 	charlieUserID := "@charlie:hs2"
 	charlie := deployment.Client(t, "hs2", charlieUserID)
 
+	// Create a server to observe
```

Comment:
Since it might not be immediately obvious, I might just state here that the purpose of this third server is to ensure that knocking events are observed by all homeservers in the room, not just those are knocking and sending the knock event into the room on behalf of the knocking server.

Reply:
It was clear to me through the use of the word "observe" in this comment.